commit 8f30dc8bd2c2d3eab259425d1352edc7efa2bf6b
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Tue Feb 6 11:48:17 2018 +0000

    BCM270X_DT: Minor cosmetic DT tidy
    
    Deleting the timer node for all bcm27* DTBs than re-adding an identical
    one for bcm2708 is wrong - just delete it where it isn't wanted.
    
    Also change a #include to match the style of similar includes.
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/boot/dts/bcm2708.dtsi b/arch/arm/boot/dts/bcm2708.dtsi
index 0f72bd90c591..d3bd39bbd5e4 100644
--- a/arch/arm/boot/dts/bcm2708.dtsi
+++ b/arch/arm/boot/dts/bcm2708.dtsi
@@ -3,14 +3,5 @@
 #include "bcm2708-rpi.dtsi"
 
 / {
-	soc {
-		timer@7e003000 {
-			compatible = "brcm,bcm2835-system-timer";
-			reg = <0x7e003000 0x1000>;
-			interrupts = <1 0>, <1 1>, <1 2>, <1 3>;
-			clock-frequency = <1000000>;
-		};
-	};
-
 	/delete-node/ cpus;
 };
diff --git a/arch/arm/boot/dts/bcm2709.dtsi b/arch/arm/boot/dts/bcm2709.dtsi
index c3aedc6541ad..715f5aca753e 100644
--- a/arch/arm/boot/dts/bcm2709.dtsi
+++ b/arch/arm/boot/dts/bcm2709.dtsi
@@ -6,6 +6,8 @@
 	soc {
 		ranges = <0x7e000000 0x3f000000 0x01000000>,
 		         <0x40000000 0x40000000 0x00040000>;
+
+		/delete-node/ timer@7e003000;
 	};
 
 	__overrides__ {
diff --git a/arch/arm/boot/dts/bcm270x.dtsi b/arch/arm/boot/dts/bcm270x.dtsi
index cf8564bbe2b0..8fba2e245a09 100644
--- a/arch/arm/boot/dts/bcm270x.dtsi
+++ b/arch/arm/boot/dts/bcm270x.dtsi
@@ -1,5 +1,5 @@
 /* Downstream bcm283x.dtsi diff */
-#include "dt-bindings/power/raspberrypi-power.h"
+#include <dt-bindings/power/raspberrypi-power.h>
 
 / {
 	chosen {
@@ -8,8 +8,6 @@
 
 	soc: soc {
 
-		/delete-node/ timer@7e003000;
-
 		watchdog: watchdog@7e100000 {
 			/* Add alias */
 		};
diff --git a/arch/arm/boot/dts/bcm2710.dtsi b/arch/arm/boot/dts/bcm2710.dtsi
index 3f2a2a259754..2671731ff440 100644
--- a/arch/arm/boot/dts/bcm2710.dtsi
+++ b/arch/arm/boot/dts/bcm2710.dtsi
@@ -29,6 +29,8 @@
 			interrupts = <9>;
 		};
 
+		/delete-node/ timer@7e003000;
+
 		timer {
 #ifdef RPI364
 			compatible = "arm,armv8-timer", "arm,armv7-timer";
